app/helpers/filter_dropdown_helper.rb

```diff
@@ -99,6 +99,11 @@ def dropdown_toggler_options
 
   def translate_value(filter_attribute, t_scope)
     return t('all') if filter_attribute.blank?
-    t(filter_attribute, t_scope)
+    unless filter_attribute.class == Department ||
+        filter_attribute.class == GroupOfferCategory ||
+        filter_attribute.to_s.to_i != 0
+      return t(filter_attribute, t_scope)
+    end
+    filter_attribute.to_s
```

Comment:
Hmm this method is getting *really* hairy, @casaper is also adding some changes in https://github.com/panter/aoz-003/pull/229/files#diff-c96b9a7dfc4caea3be8d6e40b89c9ad2

It looks like one of the issues is that `filter_attribute` is sometimes a record (`list_filter_link`), and sometimes an attribute name (`toggler_text`)?

Please try to refactor this method together with @casaper, or if it's getting too confusing just add some tests to make sure it won't break :)

Reply:
Yep, that would make sense. Or use it in the same manner as it was used before: by not passing records, just symbols and t_scope if needed.

Reply:
I think it's already passing records in the current `develop` in `list_eq_filter_dropdown` (`item`) and `list_boolean_filter_dropdown` (`collection[index]`), which become `filter_attribute` in `list_filter_link`, which then gets passed to `translate_value`.